src/misc/00_intro.rst

```diff
@@ -21,12 +21,16 @@ Glossary
   ICU
     **I**\ nternational **C**\ omponents for **U**\ nicode. ICU is an open-source project of mature C/C++ and Java libraries for Unicode support, software internationalization, and software globalization. `Learn More <http://site.icu-project.org/>`_. 
     
-    
+  R
```

Comment:
I don't see a reason for R being referred here, it's not something that appears too often in the docs and we can get the grasp of it by the context

Reply:
Thanks for the feedback. I was thinking R would be a good addition since it is mentioned in the CRAN definition. Thanks again, I'll update accordingly.